runtime/src/cost_tracker.rs

```diff
@@ -9,6 +9,43 @@ use {
     std::collections::HashMap,
 };
 
+/// Copied nightly-only experimental `checked_add_signed` implementation
+/// from https://github.com/rust-lang/rust/pull/87601/files
+/// Should be removed once the feature is added to standard library.
+trait ArithmeticUtil {
```

Comment:
I think this should live in sdk instead

Reply:
move to sdk in [commit](89397ea8d43a73a637b31ad58662a88d84b2f53a)